crates/services/importer/src/importer.rs

```diff
@@ -148,6 +154,7 @@ impl<D, E, V> Importer<D, E, V> {
             broadcast,
             active_import_results: Arc::new(Semaphore::new(max_block_notify_buffer)),
             guard: Semaphore::new(1),
```

Comment:
nit: Semaphore with limit of 1 is better expressed as `Mutex<()>`, of course assuming this is always 1.